src/native/external/libunwind_extras/CMakeLists.txt

```diff
@@ -165,12 +167,19 @@ if(CLR_CMAKE_HOST_WIN32)
     set_source_files_properties(${LIBUNWIND_SOURCES} PROPERTIES COMPILE_FLAGS /TC) # compile all files as C
 endif(CLR_CMAKE_HOST_WIN32)
 
+if(CLR_CMAKE_TARGET_ANDROID)
+    # Instead of patching libunwind, don't treat the handful of warnings it has as errors
+    add_compile_options(-Wno-error)
```

Comment:
This should be an issue. We regularly upstream changes to libunwind. See https://github.com/libunwind/libunwind/pull/734.

/cc @am11 @janvorli

Reply:
Could you show the warnings? We should fix them here instead and upstream the patch simultaneously, and add a line in `../libunwind-version.txt`.

Reply:
https://github.com/dotnet/runtime/issues/111527